### PR TITLE
About me

### PR DESCRIPTION
**Pull request by giovannifld**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ const App = () => {
     .catch(error => console.log("login errors: ", error))
   }
   return (
-    <>
+    <body>
```
Comment (SunkissedQueen) [inline src/App.js:100]: make sure to nest everything within your body tags

File: src/assets/about-me-pic.jpeg
Comment (SunkissedQueen) [inline src/assets/about-me-pic.jpeg:1]: love this pic...so much happiness

File: src/assets/aleja-pic.jpg
Comment (SunkissedQueen) [inline src/assets/aleja-pic.jpg:1]: nice pic does not look like you dish out smack talk

File: src/assets/xena-pic.jpg
Comment (SunkissedQueen) [inline src/assets/xena-pic.jpg:1]: looks like you are about to give some good advice

File: src/pages/AboutUs.js [hunk 2]
```diff
@@ -1,9 +1,125 @@
...
+          LinkedIn
+        </CardLink>
+        <CardLink className='about-us-link' href="https://github.com/alopezdelgado" target='_blank'>
+          GitHub
+        </CardLink>
+      </CardBody>
+    </Card>
+        </div>
+      </div>
+    </>
```
Comment (SunkissedQueen) [inline src/pages/AboutUs.js]: indentation

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
please see comments

File: src/assets/aleja-pic.jpg
Comment (AlejaH1213) [reply src/assets/aleja-pic.jpg:1]: I like to keep them guessing

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**